ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -1917,7 +1917,8 @@ public HiveRelOptMaterialization getMaterializedViewForRebuild(String dbName, St
             // We will not try partial rewriting if there were update/delete operations on source tables
```

Comment:
This comment is out of date, can you update it?

Reply:
updated